bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="souliss" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Souliss Binding</name>
+	<description>This is the binding for Souliss. The Arduino based SmartHome..</description>
+	<author>Tonino Fazio - Luca Remigio - Luca Calcaterra</author>
```

Comment:
I'm pretty sure I saw people comment that the author tag was removed from the spec, so please remove this line.

Reply:
fixed